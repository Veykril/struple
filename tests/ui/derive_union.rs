use struple::Struple;

#[derive(Struple)]
union Foo {
    foo: i32,
    bar: u64,
}

fn main() {}
