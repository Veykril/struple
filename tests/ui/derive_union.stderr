error: Struple may only be derived on structs
 --> $DIR/derive_union.rs:3:10
  |
3 | #[derive(Struple)]
  |          ^^^^^^^
